examples/docker-gen.cfg

```diff
@@ -8,3 +8,11 @@ notifycmd = "/etc/init.d/nginx reload"
 template = "/etc/logrotate.conf.tmpl"
 dest = "/etc/logrotate.d/docker"
 watch = true
+
+[[config]]
+Template = "/etc/docker-gen/templates/nginx.tmpl"
+Dest = "/etc/nginx/conf.d/default.conf"
+watch = true
+[config.NotifyContainers]
+nginx = 1  # 1 is a signal number to be sent; here SIGINT
```

Comment:
I think this would be more self-documenting and easier to configure if the signal was a string.  e.g.

```
nginx = "SIGINT"
```

`docker-gen` can map the string to the signal number.  I would also make the string comparison case-insensitive too.